### ISSUE TITLE
No such option error after installation

### ISSUE THREAD

**Issue by rragundez** [seebk/LaTeXText#21]
When trying to render I get

`latextext.py: error: no such option: --id`

![selection_009](https://user-images.githubusercontent.com/16643700/47788095-069dfd80-dd11-11e8-81d5-ba435e21b4a7.png)

**Comment by seebk** [comment]
That's weird, never saw this error and no idea where this "--id" option is coming from. Where did you download the extension and what OS and Inkscape version do you use?

**Comment by rragundez** [comment]
I cloned the repo and ran the `install.sh`, I did have to change the path though, because the Inkscape extensions directory is in another place as shown below

![selection_011](https://user-images.githubusercontent.com/16643700/48047539-9a9c1900-e198-11e8-878b-b431c5afa1e3.png)


OS: Ubuntu 18.04
Inkscape: 0.94

**Comment by seebk** [comment]
Ok, I see... the Snap package uses a different path prefix for everything. Not ideal when it should work together with other tools on the same system. You could manually copy the extension files to /home/rodrigo/snap/inkscape/4274/... Somewhere there should be an extensions subfolder below this path.

Is Inkscape 0.9.4 the current development version? Maybe the extension format changed and now requires this new `--id` parameter?

**Comment by rragundez** [comment]
Yeah so I changed the paths in the `install.sh` and it does appear in my extensions inside inkscape. The version that I have is actually 0.92

![selection_012](https://user-images.githubusercontent.com/16643700/48054548-d7263f80-e1ad-11e8-8a71-30a66b3c065a.png)

**Comment by coschroeder** [comment]
Hi,
I have the same problem, with the same inkscape version, but ubuntu 16.4.
I changed the paths in the install.sh but I still get the error 
`latextext.py: error: no such option: --id`
is there something else that I have to change?

**Comment by seebk** [comment]
That's weird, I still cannot reproduce the issue here.... :(
